mvp.feature

```diff
@@ -896,11 +942,149 @@ Feature: MVP features for NVA
+    And the request is deleted from the work list of their curator
+    And the Public Page for Publication has an enabled "Request DOI" button
+
+  @1242
+  Scenario: Owner navigates to the Public Page for Publication for a Publication with an existing DOI
+    Given that the Creator navigates to the Public Page for Publication
+    And they are the Owner of this Publication
+    And the Publication has a DOI
+    When they see the Status Bar
+    Then they see that the Request/Draft DOI button is disabled
+  # a Publication may have more then one DOI,
+  # but NVA is only source for one DOI for each Publication
+
+  @1231
+  Scenario: Owner navigates to the Public Page for Publication for published publication without DOI
+    Given that the Creator navigates to the Public Page for Publication
+    And they are the Owner of this Publication
+    And the Publication has no DOI
+    When they see the Status Bar
+    Then they see buttons for Request a DOI and Edit Publication
+
+  @511
+  Scenario: Owner navigates to the Public Page for Publication of a Publication and opens the "Request a DOI" dialog
+    Given that a Creator navigates to the Public Page for Publication for published publication without DOI
+    And they are the Owner of this Publication
+    When they click the "Request a DOI" button
+    Then the "Request a DOI dialog" is opened
+    And they see fields for Message
+    And a button for Send Request
+
+  @1232
+  Scenario: Owner navigates to the Public Page for Publication and requests a DOI
+    Given that the Creator navigates to the Public Page for Publication
+    And they are the owner of the publication
+    And open "Request a DOI" dialog
+    And optional add a message to the curator
+    When the user click the send button
+    Then the Public Page for Publication is displayed
+    And the "Request a DOI" button is renamed to "DOI pending" and is disabled
+    And the request is listed in user work list
+    And the request is listed in curator work list
+
+  @1233
+  Scenario: Owner navigates to the Public Page for Publication for unpublished publication without DOI
+    Given that the Owner view Public Page for Publication for publication
+    And the publication is not Published
+    And the publication has no DOI
+    When they look at the Status Bar
+    Then they see buttons for Draft a DOI and Edit publication
+
+  @1234
+  Scenario: Owner navigates to the Public Page for Publication and selects Draft a DOI
+    Given that the Owner View Public Page for Publication for unpublished publication without DOI
+    And they are the owner of the Publication
+    When they click the "Draft a DOI" button
+    Then the Public Page for Publication is displayed
+    And the "Draft a DOI" button is renamed to "DOI pending" and is disabled
+    And the Draft DOI is added to the metadata
+    And the Public Page for Publication contains the Draft DOI
+    And the Draft DOI is not a link
```

Comment:
There should be a scenario or a description of what a Draft DOI is because we only now what a Draft DOI is not.